starter/src/main/resources/application.properties

```diff
@@ -25,3 +25,6 @@ spring.kafka.consumer.value-deserializer= org.apache.kafka.common.serialization.
 spring.kafka.consumer.properties.spring.json.trusted.packages=*
 
 
+spring.security.oauth2.client.registration.facebook.client-id=579437707268071
+spring.security.oauth2.client.registration.facebook.client-secret=7656e20008799392ba997c4a6cb3863b
```

Comment:
Should we commit this into a public repository on github? :-)

Reply:
Definetly not. Going to fix